src/main/resources/CU-ApplicationResources.properties

```diff
@@ -248,7 +248,7 @@ error.po.not.fully.received.and.invoiced=PO indicates receiving is required and
 
 	errors.add.new.note.sendToVendor.noAtt=Send to Vendor note must have an Attachment.
 	errors.exceed.sq.number.of.att.limit=Failed to {0} because the number of 'Send to Vendor' Attachments exceeds Sciquest limit of {1}. 
-	errors.att.filesize.over.limit=Attachment "{0}" file size over limit of {1} MB.
```

Comment:
As I noted in my previous review, this old property key was defined in two different spots within this file. If you end up removing the property key entirely, make sure you remove its other reference from this file.